### PR TITLE
Add privacy features polkadot extension

### PR DESCRIPTION
**Pull request by jonasW3F**
### Project Abstract

A RFP to add privacy features to the Polkadot extension.

### For which grant [level](https://github.com/w3f/Grants-Program#level_slider-levels) are you applying? 
- [X] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $50,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for > $100k Web3 Foundation Council approval)

### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( `project_name.md`) and updated.
- [ ] A BTC or Ethereum (DAI/USDT) address for the payment of the milestones is provided inside the application.
- [ ] I have read and acknowledged the [terms and conditions](https://github.com/w3f/Grants-Program/blob/master/docs/T&Cs.md).
- [ ] The software delivered for this grant will be released under an open-source license specified in the application.
- [ ] The initial PR contains only one commit (squash and force-push if needed).
- [ ] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process).

### CODE REVIEW

File: rfps/privacy-enhancement-polkadot-extension.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Privacy Enhancement for Polkadot Extension
+
+* **Status:** Open
+* **Proposer:** jonasW3F
+* **Your Project(s):** [optional]: N/A
```
Comment (mmagician) [inline rfps/privacy-enhancement-polkadot-extension.md:5]:
```suggestion
```

File: rfps/privacy-enhancement-polkadot-extension.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* **Projects you think this work could be useful for** [optional]: https://github.com/polkadot-js/extension
+* **Teams/People that could deliver the RFP** [optional]: N/A
```
Comment (mmagician) [inline rfps/privacy-enhancement-polkadot-extension.md:7]:
```suggestion
```

File: rfps/privacy-enhancement-polkadot-extension.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+* **Total Estimated Duration:** 1 month
+* **Full-time equivalent (FTE):**  Amount of time (in days) required for a single person to complete this project ([see](https://en.wikipedia.org/wiki/Full-time_equivalent)) 
+* **Total Costs:** $10'000. 
+
+### Milestone 1
+ 
+* **Estimated Duration:** 1 month
+* **FTE:**  Amount of time (in days) required for a single person to complete this milestone
+* **Costs:** $10'000
```
Comment (mmagician) [inline rfps/privacy-enhancement-polkadot-extension.md:25]:
```suggestion
* **Costs:** DAI 10,000
```

**Review by mmagician — APPROVED** [review]

**Comment by mmagician** [comment]
Thanks @jonasW3F, looks great!

File: rfps/privacy-enhancement-polkadot-extension.md (hunk 3 above)
Comment (alxs) [reply rfps/privacy-enhancement-polkadot-extension.md:25]: I'd actually leave this, pricing should be in USD while payment can be in any currency

File: rfps/privacy-enhancement-polkadot-extension.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+* **Status:** Open
+* **Proposer:** jonasW3F
+* **Your Project(s):** [optional]: N/A
+* **Projects you think this work could be useful for** [optional]: https://github.com/polkadot-js/extension
+* **Teams/People that could deliver the RFP** [optional]: N/A
+
+## Project Description :page_facing_up: 
+
+An increasing number of websites require access to the Polkadot extension with a growing ecosystem. By allowing access to the extension, websites (naturally) can see the addresses that are contained in the extension. This imposes a big risk to privacy, because malicious actors could create lists about which addresses belong to one entity and, in the worst case, even link it to real-world identities (if at least one address is linked to an on-chain identity). A feature to prevent this is currently the "hide" button on single addresses, which prevent websites from seeing that address. However, it is currently cumbersome to handle that feature. The idea of this RFP is to extend on that feature and include a few QOL functionalities to make it easier for users to protect their privacy.
```
Comment (alxs) [inline rfps/privacy-enhancement-polkadot-extension.md:11]: I actually don't have a 'hide' button, which browser are you using?

Comment (semuelle) [reply rfps/privacy-enhancement-polkadot-extension.md:11]: Every account has an eye symbol on the right side, which you can click. It's at least on Chrome and Firefox. Unless your extension is really outdated. ;)

Comment (jonasW3F) [reply rfps/privacy-enhancement-polkadot-extension.md:11]: yes, you should be able to hide individual accounts already

File: rfps/privacy-enhancement-polkadot-extension.md (hunk 2 above)
Comment (mmagician) [reply rfps/privacy-enhancement-polkadot-extension.md:7]: Actually, maybe @celrisen could be interested

**Review by mmagician — APPROVED** [review]

**Review by semuelle — APPROVED** [review]

**Pull request merged**